### DISCUSSION TITLE
Foocus Gradio Not showing Public Link

### DISCUSSION THREAD

**Discussion by Tranx-ui** [lllyasviel/Fooocus#2820] [Q&A] [answered]
When I Start the fooocus in Google collab it does not give me public link 
It says download some file and paste it 
And if I do that it doesn't work 
What to do?

**Comment by poor7** [comment, marked as answer]
Duplicated
https://github.com/lllyasviel/Fooocus/discussions/2818

https://github.com/lllyasviel/Fooocus/issues/2819
https://github.com/lllyasviel/Fooocus/issues/2817
https://github.com/lllyasviel/Fooocus/issues/2815
